src/DataPersister/RemoveStamp.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Feels like this should rather go into `DataPersister/Messenger/RemoveStamp`. It's not a data persister itself. Wdyt?

Reply:
Following symfony hierarchy I'd even say Stamp/RemoveStamp

Reply:
src/Bridge/Symfony/Messenger ^^